src/services/domain/memory-bank.service.ts

```
@@ -0,0 +1,154 @@
+    repositoryProvider: RepositoryProvider,
+    getKuzuClient: (
+      mcpContext: EnrichedRequestHandlerExtra,
+      clientProjectRoot: string,
+    ) => Promise<KuzuDBClient>,
+    getSnapshotService: (
+      mcpContext: EnrichedRequestHandlerExtra,
+      clientProjectRoot: string,
+    ) => Promise<SnapshotService>,
+  ) {
+    super(repositoryProvider, getKuzuClient, getSnapshotService);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)